cypress/e2e/join_must_confirm/index.cy.ts

```diff
@@ -2,7 +2,9 @@
 
 describe("join organizations", () => {
   it("join big company with free email provider", function () {
-    cy.login("unused1@yopmail.com", "password123");
+    cy.visit(`/users/start-sign-in`);
```

Comment:
I have a question, why do I sometimes see "cy.visit(`/connection-and-account`);" and sometimes "cy.visit(`/users/start-sign-in`);"?